src/fundamental/iovec-util-fundamental.h

```diff
@@ -31,6 +31,16 @@ static inline void iovec_done(struct iovec *iovec) {
         iovec->iov_len = 0;
 }
 
+static inline void iovec_done_many(struct iovec iovec[], size_t n) {
```

Comment:
call this iovec_done_many_and_free a la close_many_and_free?

also, I think we should take pointer rather than [] here. I know it is essentially the same, but we free() it below (again matches what we do in close_many_and_free)